apps/api/drizzle/0019_short_jane_foster.sql

```diff
@@ -0,0 +1,25 @@
+ALTER TABLE "agent_rank" RENAME TO "agent_score";--> statement-breakpoint
+ALTER TABLE "agent_rank_history" RENAME TO "agent_score_history";--> statement-breakpoint
+ALTER TABLE "agent_score" DROP CONSTRAINT "unique_agent_rank_agent_id";--> statement-breakpoint
+ALTER TABLE "agent_score" DROP CONSTRAINT "agent_rank_agent_id_fkey";
+--> statement-breakpoint
+ALTER TABLE "agent_score_history" DROP CONSTRAINT "agent_rank_history_agent_id_fkey";
+--> statement-breakpoint
+ALTER TABLE "agent_score_history" DROP CONSTRAINT "agent_rank_history_competition_id_fkey";
+--> statement-breakpoint
+ALTER TABLE "object_index" ALTER COLUMN "data_type" SET DATA TYPE text;--> statement-breakpoint
+DROP TYPE "public"."sync_data_type";--> statement-breakpoint
```

Comment:
```suggestion
UPDATE "object_index" SET "data_type" = 'agent_score_history' WHERE "data_type" = 'agent_rank_history';--> statement-breakpoint
UPDATE "object_index" SET "data_type" = 'agent_score' WHERE "data_type" = 'agent_rank';--> statement-breakpoint
DROP TYPE "public"."sync_data_type";--> statement-breakpoint
```

ah _now_ i remember the problem you mentioned about scripts and backfilling data. 

try this and see if it works. it seemed to fix the migration problem for me locally:
1. checkout `main` and seed the db with at least 1 "ended" comp
2. checkout this branch, and run the migration w/o these changes...this will error with `applying migrations...error: invalid input value for enum sync_data_type: "agent_rank_history"`
3. make the change above, and the migration works.